.github/CODEOWNERS

```diff
@@ -1 +1,3 @@
-/packages/api @favna @QuantumlyTangled
+* @favna
```

Comment:
?

Reply:
it was like that when I inherited the branch, haven't touched it.

Reply:
![image](https://user-images.githubusercontent.com/22531310/103444006-8828a780-4c5c-11eb-950a-19b25be3e3e0.png)
Can confirm, blame reveals it was commits prior to mine a whole 8 days ago that made that change. I have no idea.

Reply:
Weird, well thanks for fixing it.